### DISCUSSION TITLE
Is there a way to increase field value on Update Data Operation?

### DISCUSSION THREAD

**Discussion by yisheng** [directus/directus#16627] [Archive]
Could Update Data Operation have this kind of ability:

```sql
UPDATE content SET count = count + 1 WHERE id = xxxx
```

[On this thread](https://github.com/directus/directus/discussions/3147), Ben said `it's certainly something that can already be performed with the API`, but I can't find how to do it in the docs.

Thanks!

**Comment by br41nslug** [comment]
This is not currently a feature in the api. The way to do that is to fetch the data and save the incremented value either on the frontend or in a custom endpoint/flow
